### PR TITLE
c38 translation update

### PR DESCRIPTION
**Pull request by ManlyMarco**
Thanks to @ScrewThisNoise for providing the translations.

### CODE REVIEW

File: translation/en/RedirectedResources/assets/abdata/h/list/50_00/personality_voice_c38_02_00/translation.txt [hunk 1]
```diff
@@ -484,5 +484,5 @@
 んぅっ、貴方が元気過ぎて……私の方が、先にまいってしまいそうです……=Nnnn, you're way too energetic... I feel as though I'll end up run out of steam first...
 全部、飲むこともできるのですよ？　ですが、貴方は見たいのですよね？=I can drink it all too, you know? However, you want to appreciate this sight, don't you?
 はい、お疲れ様でした……ふふ、終わりの挨拶には、まだ早いですか？=Alright, great job... fufuh, is it too soon to close it off like that, still?
-（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufu--, doing it in the pool, you really come up with quite the lewd stuff sometimes.
+（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufuh--, in the pool, you truly do come up with some lewd stuff.
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/h/list/50_00/personality_voice_c38_02_00/translation.txt:487]:
```suggestion
（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufu, in the pool, you truly do come up with some lewd stuff.
```
or
```suggestion
（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufu--  In the pool, you truly do come up with some lewd stuff.
```

File: translation/en/RedirectedResources/assets/abdata/h/list/50_00/personality_voice_c38_03_00/translation.txt [hunk 2]
```diff
@@ -1054,7 +1054,7 @@
 お尻の奥まで、見られてしまいましたね……=You saw all the way deep inside my anal depths, didn't you...
 お尻にほら、こんなに出して……=You came so much inside, see?...
 はぁぁっ、大きいのも一緒に出てしまいそうです……=Haaah--, I feel like I'm going to end up letting out a big one together with it...
-（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufu--, doing it in the pool, you really come up with quite the lewd stuff sometimes.
+（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufuh--, in the pool, you truly do come up with some lewd stuff.
```
Comment (GeBo1) [inline translation/en/RedirectedResources/assets/abdata/h/list/50_00/personality_voice_c38_03_00/translation.txt:1057]:
```suggestion
（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufu, in the pool, you truly do come up with some lewd stuff.
```
or
```suggestion
（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufu--  In the pool, you truly do come up with some lewd stuff.
```

**Review by GeBo1 — CHANGES_REQUESTED** [review]

File: translation/en/RedirectedResources/assets/abdata/h/list/50_00/personality_voice_c38_02_00/translation.txt (hunk 1 above)
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/h/list/50_00/personality_voice_c38_02_00/translation.txt:487]:
```suggestion
（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufu-- Doing it in the pool, you truly do come up with quite the lewd stuff sometimes.
```

File: translation/en/RedirectedResources/assets/abdata/h/list/50_00/personality_voice_c38_03_00/translation.txt (hunk 2 above)
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/h/list/50_00/personality_voice_c38_03_00/translation.txt:1057]:
```suggestion
（開始時台詞）ふふっ、プールの中でって、本当にエッチなこと考えるわね=Fufu-- Doing it in the pool, you truly do come up with quite the lewd stuff sometimes.
```

**Pull request merged**